ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -107,6 +109,24 @@ export class AptosAccount {
     return authKey.derivedAddress();
   }
 
+  /**
+   * Takes source address and seeds and returns the resource account address
+   * @param sourceAddress Address used to derive the resource account
+   * @param seed The seed bytes
+   * @returns The resource account address
+   */
+
+  static getResourceAccountAddress(sourceAddress: MaybeHexString, seed: Uint8Array): HexString {
```

Comment:
does the user expected to provide the seed? should we add some documentation around this? like how one can get the seed?

Reply:
No need. seed is whatever they used the create the resource account in Move